src/actions/favoriteGifs.js

```diff
@@ -0,0 +1,85 @@
+const GET_FAVORITE_GIFS_REQUESTED = 'GET_FAVORITE_GIFS_REQUESTED';
+const GET_FAVORITE_GIFS_COMPLETED = 'GET_FAVORITE_GIFS_COMPLETED';
+const GET_FAVORITE_GIFS_FAILED = 'GET_FAVORITE_GIFS_FAILED';
+const ADD_FAVORITE_GIF_REQUESTED = 'ADD_FAVORITE_GIF_REQUESTED';
+const ADD_FAVORITE_GIF_COMPLETED = 'ADD_FAVORITE_GIF_COMPLETED';
+const ADD_FAVORITE_GIF_FAILED = 'ADD_FAVORITE_GIF_FAILED';
+const REMOVE_FAVORITE_GIF_REQUESTED = 'REMOVE_FAVORITE_GIF_REQUESTED';
+const REMOVE_FAVORITE_GIF_COMPLETED = 'REMOVE_FAVORITE_GIF_COMPLETED';
+const REMOVE_FAVORITE_GIF_FAILED = 'REMOVE_FAVORITE_GIF_FAILED';
+
+const getFavoriteGifsRequested = (pagination) => ({
+  type: GET_FAVORITE_GIFS_REQUESTED,
+  payload: pagination
+});
+
+const getFavoriteGifsCompleted = gifs => ({
+  type: GET_FAVORITE_GIFS_COMPLETED,
+  payload: gifs,
+});
+
+const getFavoriteGifsFailed = error => ({
+  type: GET_FAVORITE_GIFS_FAILED,
+  error,
+});
+
+
+const addFavoriteGifRequested = (gif) => ({
```

Comment:
Action creators should wrap the payload inside an object:

```
const addFavorite = (gif) => (
  type: 'ADD_FAVORITE_GIF',
  payload: { gif },
);
```
This is in order to have a standard way of transporting payloads, and
comply with the Flux standard.

Reply:
Ok ok i got it, saving

Reply:
I changed all similar cases too 👊, thank you so much for your suggestion 👍🤝